src/Model/BunqModel.php

```diff
@@ -223,43 +221,26 @@ protected static function classFromJson($class, BunqResponseRaw $responseRaw)
     protected static function processForId(BunqResponseRaw $responseRaw)
     {
         /** @var Id $id */
-        $id = Id::fromJson($responseRaw)->getValue();
+        $id = Id::fromJson($responseRaw, self::FIELD_ID)->getValue();
 
         return new BunqResponse($id->getId(), $responseRaw->getHeaders());
     }
 
     /**
      * @param BunqResponseRaw $responseRaw
+     * @param string|null $wrapper
      *
      * @return BunqResponse
      * @throws BunqException   When the result is not expected.
      */
-    protected static function fromJson(BunqResponseRaw $responseRaw)
+    protected static function fromJson(BunqResponseRaw $responseRaw, string $wrapper = null)
```

Comment:
Strictly typing the return value will make the PHPDoc unnecessary.

Reply:
@DennisSnijder there are some things which, as of PHP 7.0 (and partially 7.1), can be expressed with PHPDoc, but not with the strict typing of PHP. Examples are nullable return  statements (PHP 7.1+),`throws` (not present even in PHP 7.1) or late static binding.

Therefore, for now we would like to keep it to PHPDoc for now, until PHP has full and proper support of the strict typing :).